swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+  offset:
+    name: offset
+    description: The first row of resources to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter.
+    type: integer
+    required: false
+    in: query
 
 definitions:
-  HelloResponse:
+  BulletinsList:
+    type: object
+    properties:
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Bulletin'
+
+  Bulletin:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      contacts:
+        type: array
+        items:
+          $ref: '#/definitions/Contact'
+      description:
+        $ref: '#/definitions/summary'
+      keywords:
+        type: array
+        description: List of keywords relating to this bulletin
+        items:
+          type: string
+        example: [ "work" , "jobless" ]
+      license:
+        description: "The standard Government license right text for the bulletin"
+        type: string
+      national_statistic:
+        $ref: '#/definitions/nationalStatistic'
+      next_release:
+        $ref: '#/definitions/nextRelease'
+      related_links:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_data:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_documents:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_methodologies:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_methodology_articles:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_topics:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      title:
+        $ref: '#/definitions/title'
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          editions:
+            $ref: '#/definitions/Link'
+          latest_edition:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+          latest_version:
+            $ref: '#/definitions/Link'
+
+  EditionsList:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Edition'
+
+  Edition:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          bulletin:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+          latest_version:
+            $ref: '#/definitions/Link'
+
+
+  VersionsList:
+    type: object
+    properties:
```

Comment:
We should probably add the following fields:
* `limit`: limit requested
* `offset`: offset requested
* `total_count`: total number of bulletins